packages/flutter_tools/test/general.shard/web/devfs_web_test.dart

```diff
@@ -150,6 +150,20 @@ void main() {
     Platform: () => windows,
   }));
 
+   test('serves asset files from in filesystem with url-encoded paths', () => testbed.run(() async {
+    final File source = globals.fs.file(globals.fs.path.join('build', 'flutter_assets', Uri.encodeFull('abcd象形字.md')))
```

Comment:
extreme nit: maybe change the extension to `.png` instead of `.md`

Reply:
Done